pyomo/contrib/doe/doe.py

```diff
@@ -309,6 +355,38 @@ def run_doe(self, model=None, results_file=None):
         model.objective.activate()
         model.obj_cons.activate()
 
+        if self.use_grey_box:
+            # Initialize grey box inputs to be fim values currently
+            for i in model.parameter_names:
+                for j in model.parameter_names:
+                    if list(model.parameter_names).index(i) >= list(
+                        model.parameter_names
+                    ).index(j):
+                        model.obj_cons.egb_fim_block.inputs[(j, i)].set_value(
+                            pyo.value(model.fim[(i, j)])
+                        )
+                    else:  # REMOVE THIS IF USING LOWER TRIANGLE
+                        pass
+                        # model.obj_cons.egb_fim_block.inputs[(j, i)].set_value(
+                        #    pyo.value(model.fim[(j, i)])
+                        # )
```

Comment:
This confuses me - is this leftover code / note that's not supposed to be here anymore?